generators/app/templates/src/containers/Profile/profile.test.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { Profile } from "./profile.container";
+import { ProfileComponent } from './profile.component';
+import data from "@solid/query-ldflex";
```

Comment:
Not using this, right? Might want to have a jslint config protecting you from that.

Reply:
yes I have it but I don't know why not show it, i will check it thank you !!!